src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Constants/LoggingConstants.cs

```diff
@@ -8,5 +8,7 @@ internal static class LoggingConstants
         public const string ZeroIpAddress = "0.0.0.0";
         public const string Unknown = "[Unknown]";
         public const string ClientIpKey = "ClientIp";
+        public const string HostDiagnosticSourcePrefix = "Microsoft.Azure.Functions.Host.";
```

Comment:
Since this is taking advantage of a host feature... it seems to me like this code should be in the host. It'd be a lot simpler to test e2e as well as you wouldn't need to make an update, then consume the package, etc.

Reply:
I will close this PR and move the logic to Host